app/src/components/mainWindow/mainWindow.js

```diff
@@ -89,6 +89,8 @@ function createMainWindow(inpOptions, onAppQuit, setDockBadge) {
     icon: getAppIcon(),
     // set to undefined and not false because explicitly setting to false will disable full screen
     fullscreen: options.fullScreen || undefined,
+    // window modal defaulted to false
```

Comment:
Unnecessary comment, and "modal" means something different than "always on top", pls remove

Reply:
sure yes.

Reply:
PR updated! Thanks.